### ISSUE TITLE
Helm chart

### ISSUE THREAD

**Issue by data-dude** [tremor-rs/tremor-www-docs#175]
It would be nice if there was a Helm chart for getting started on Kubernetes.

**Comment by Licenser** [comment]
Heya @data-dude,
would https://github.com/tremor-rs/tremor-k8s-helm help you?

**Comment by data-dude** [comment]
Yup Thanks

**Comment by Licenser** [comment]
I'll re-open it and rephrase it to mention the helm chart :) if you couldn't find it from the docs it's a bug on our side.

**Comment by afzal442** [comment]
Hi there,
Lemme know if this issue exists anymore. If yes, I want to work on it. Thanks

**Comment by data-dude** [comment]
I"m fine because I saw the link posted by Licenser.   You might want to consider putting that link on this page with a K8s section: https://docs.tremor.rs/development/quick-start/

**Comment by skoech** [comment]
Good idea, @data-dude. We will add a note on that in the 'quick start' page. Thanks! :)

**Comment by skoech** [comment]
@afzal442, you are welcome to work on this. Feel free to ask any question or for any clarification. :)

**Comment by afzal442** [comment]
Ok cool! As I am new here, I really need your supports on the fly. I went through the docs first. 

I think it will be my entry point to get started with this project.

**Comment by skoech** [comment]
Great place to start! Let me know how you would like to progress from there whenever you're ready.

**Comment by afzal442** [comment]
Thanks.
> We will add a note on that in the 'quick start' page.

I think I can proceed like this as below to add in the below section or you can suggest here:

_**Note**_: To know about _Helm chart_ for getting started with Tremor for Kubernetes. Please find the project from[ here](https://github.com/tremor-rs/tremor-k8s-helm).

**Comment by skoech** [comment]
I think maybe you'll need to add the `tremor-k8s-helm` repo into `docs` first. What do you think?
